demisto_sdk/commands/validate/tests/PB_validators_test.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+from demisto_sdk.commands.validate.validators.PB_validators.PB118_is_input_key_not_in_tasks import (
+    IsInputKeyNotInTasksValidator,
+)
+from demisto_sdk.commands.validate.tests.test_tools import create_playbook_object
+
+
+playbook = create_playbook_object()
```

Comment:
Let's create the test items inside the paramatrize.

Reply:
Done